### PR TITLE
Nettles

### PR DESCRIPTION
**Pull request by Elijahrane**
## About the PR 
This adds nettles, with their basic features added. You can hit people and make them itchy, or grind them up to get the histamine directly. They don't get super nasty until around high potency, and it takes at least 2 nettles at max potency to inject enough histamine to overdose.

This does *NOT* add:
Death Nettles (Need to playtest injection and think of good reagents)
Botanist's Glove Restriction (The system is almost done but I am waiting for the current hand system refactor to finish)
**Screenshots**

nettlespic

**Changelog**

:cl: Rane
- add: Added nettles! They're an amazing source of histamine when cultivated to high potency or ground up.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 1]
```diff
@@ -102,6 +102,39 @@
...
+      food:
+        reagents:
+        - ReagentId: Histamine
+          Quantity: 25
+  - type: Produce
+    seed: nettle
+  - type: MeleeChemicalInjector
+    transferAmount: 6 #To OD someone you would need 2 nettles and about 6-7 hits, the DOT is likely to crit them if they are running away with almost no health
+  - type: Extractable
+    grindableSolutionName: "food"
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:132]: grindable doesn't match the solution

File: Resources/Prototypes/Entities/Clothing/Hands/gloves.yml [hunk 2]
```diff
@@ -94,6 +94,8 @@
   - type: Clothing
     sprite: Clothing/Hands/Gloves/leather.rsi
     HeatResistance: 1400
+#  - type: DangerousPickUpProtection
+#    protectionType: plant
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Hands/gloves.yml:98]: Should remove rather than add placeholder comments

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml (hunk 1 above)
Comment (Elijahrane) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:132]: strangely grinding works, but I should just remove the quotes?

**Comment by Elijahrane** [comment]
@metalgearsloth I have made the requested changes and verified in-game that they do not break grinding or injection.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**